.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
```

Comment:
how about manual mode with workflow_dispatch?

Reply:
ok, will be added in the next PR

Reply:
By the way, if it is assumed that jooby-gui can be updated without making a commit and running tests after that, then in this way a possible bug will be available to users, which would not happen if the tests were run on push/deploy